### PR TITLE
adding image crop for black border

### PR DESCRIPTION
**Pull request by sidharthsapru**

### CODE REVIEW

**Comment by sidharthsapru** [comment]
@pkulak 
Please review. The code adds image crop function to nibbler to test cropping borders

File: core/src/test/java/com/vevo/nibbler/ImageCropTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    private BufferedImage loadImage(String path) {
+        try {
+            return ImageIO.read(new File(path));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+}
```
Comment (pkulak) [inline core/src/test/java/com/vevo/nibbler/ImageCropTest.java]: Am I reading this wrong, or do this tests check to see that nothing has happened? Should we have some positive tests as well?

Comment (sidharthsapru) [inline core/src/test/java/com/vevo/nibbler/ImageCropTest.java]: First test is  border along Height got cropped. Second is width border got cropped. Third is negative test where it shouldn't touch images with no border
I wanted to have one more test  where both height and width borders are getting cropped but couldn't find a proper image

Comment (pkulak) [inline core/src/test/java/com/vevo/nibbler/ImageCropTest.java]: haha, my mistake. I didn't notice the assertNotEquals. :/

**Pull request merged**